frontend/src/app/profile/page.js

```diff
@@ -0,0 +1,104 @@
+// import { useState } from 'react';
+import Sidebar from '../components/sidebar';
+import Image from 'next/image';
+import './profile.css';
+
+
+export default function Profile() {
+//   const [profilePic, setProfilePic] = useState('/profile-pic.jpg'); // Default profile picture
+//   const [view, setView] = useState('posts'); // State to manage selected view
+
+//   const handleImageChange = (event) => {
+//     const file = event.target.files[0];
+//     if (file) {
+//       const reader = new FileReader();
+//       reader.onloadend = () => {
+//         setProfilePic(reader.result);
+//       };
+//       reader.readAsDataURL(file); // Convert image to base64 URL
+//     }
+//   };
+
+  return (
+    <div style={{ display: 'flex', backgroundColor:'#37474f'}}>
+      <Sidebar />
+      <div className="content">
+        <header className="header">
+          <h1>SearchBar?</h1>
+        </header>
+        <main className="body">
+          <div className="profileInfo">
+            <label htmlFor="profilePicInput" className="profileImageLabel">
+              <Image
+                src="/spork.png"
+                alt="Profile Picture"
+                width={200}
+                height={200}
+                className="profileImage"
+              />
+            </label>
+            <input
+              type="file"
+              id="profilePicInput"
+              accept="image/*"
+            //   onChange={handleImageChange}
+              className="profilePicInput"
+              style={{ display: 'none' }}
+            />
+            <div className="userDetails">
+              <div className='sectioncount'>
+                <div className='counter'>
```

Comment:
This is a nitpick, but is there a reason you use single quotes ' for some classNames while using double quotes " for others?

Reply:
This is just an error on my part, I was planning on just single quotes but I guess I accidentally switched quotes in some places, thanks